src/Libraries/Microsoft.Extensions.EnumStrings/README.md

```diff
@@ -1 +1,80 @@
-README
+# Microsoft.Extensions.EnumStrings
+
+It's often necessary to convert a C# enum value to a string. The standard ToString() is normally used for this, but that can not be the best in terms of allocations or performance. This package introduces a source generator that is triggered by an attribute which produces an extension method for each enum type annotated with the attribute which can efficiently convert an enum value to a string.
+
+## Install the package
+
+From the command-line:
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.EnumStrings
+```
+
+Or directly in the C# project file:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.Extensions.EnumStrings" Version="[CURRENTVERSION]" />
+</ItemGroup>
+```
+
+## Usage Example
+
+### Adding the attribute to an enum
+
+To use this feature, you can annotate an enum value with the `EnumStringAttribute` attribute. For example:
+
+```csharp
+using Microsoft.Extensions.EnumStrings;
```

Comment:
@geeknoid should we consider moving the EnumStrings attribute to the usual `System.Runtime.CompilerServices` namespace where we put the rest of the attributes in .NET in favor of a future move of this down to dotnet/runtime?

Reply:
cc: @terrajobst @stephentoub

Reply:
We certainly can.